packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -114,22 +115,22 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	public orderProperties?: string[];
 
 	/**
-	 * Array of StarkColumnProperties objects which define the columns of the data table.
+	 * Whether to display the pagination component
 	 */
 	@Input()
-	public columnProperties: StarkTableColumnProperties[] = [];
+	public paginate: boolean = false;
 
 	/**
-	 * StarkActionBarConfig object (see StarkActionBarConfig docs).
+	 * Configuration object for embedded pagination component (see StarkPaginationConfig docs).
 	 */
 	@Input()
-	public tableRowsActionBarConfig: StarkActionBarConfig;
+	public paginationConfig: StarkPaginationConfig;
 
 	/**
-	 * Output event emitter that will emit the array of columns selected (column id's).
+	 * StarkActionBarConfig object (see StarkActionBarConfig docs).
```

Comment:
You can add the link here instead of the "see xxxx docs" parts: ` {@link StarkActionBarConfig} object for the action bar component to be displayed in all the rows`

Reply:
done